src/theory/arith/kinds

```diff
@@ -133,4 +133,15 @@ nullaryoperator PI "pi"
 
 typerule PI ::CVC4::theory::arith::RealNullaryOperatorTypeRule
 
+
+constant IAND_OP \
+	::CVC4::IntAnd \
+	"::CVC4::UnsignedHashFunction< ::CVC4::IntAnd >" \
+	"util/iand.h" \
```

Comment:
Can you document somewhere the precise semantics, especially for cases where the arguments are negative or too big?

Reply:
Done.

Reply:
Thanks. I guess it is fixed now, modulo the fact that I did not find a specification of INT_TO_BITVECTOR (specifically for the case of negatives/out-of-bounds). If there is one, can you send a link? If there is none, maybe this can be added in a separate PR?

Reply:
It's here: 
https://github.com/CVC4/CVC4/pull/4668/commits/a76e594560440f683092f09f2cfe0f92a5601127#diff-63290c50fbea42b02ed8329a2666682bR136
The semantics specified here covers all integers i1, i2.

Reply:
yes, but it relies on an understanding of `int2bv` operator, this is the one I was asking about.